backend/tests/clubs/test_ticketing.py

```diff
@@ -389,6 +389,26 @@ def test_remove_from_cart_extra(self):
         cart = Cart.objects.get(owner=self.user1)
         self.assertEqual(cart.tickets.count(), 0, cart.tickets)
 
+    def test_delete_event_with_claimed_tickets(self):
+        # Simulate checkout (hold ticket)
+        self.tickets1[0].holder = self.user1
+        self.tickets1[0].save()
+
+        self.client.login(username=self.user1.username, password="test")
+        resp_held = self.client.delete(
+            reverse("club-events-detail", args=(self.club1.code, self.event1.pk))
+        )
+        self.assertEqual(resp_held.status_code, 400, resp_held.content)
+
+        # Simulate purchase (transfer ticket)
+        self.tickets1[0].owner = self.user1
```

Comment:
nit: can you set the holder to `None` here?

Reply:
we love the attention to detail, thanks for catching